### PR TITLE
fix: adjust actions layout on mobile, uploading video

### PR DESCRIPTION
**Pull request by swkatmask**
https://mask.atlassian.net/browse/fw-582
https://mask.atlassian.net/browse/fw-591

### CODE REVIEW

File: src/components/Actions/Comment.tsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ export const Comment = memo<CommentProps>(function Comment({
 
     return (
         <ClickableArea
-            className={classNames('flex cursor-pointer items-center space-x-2', {
+            className={classNames('flex cursor-pointer items-center sm:space-x-1 md:space-x-2', {
```
Comment (guanbinrui) [inline src/components/Actions/Comment.tsx:63]: sm = match all

```suggestion
            className={classNames('flex cursor-pointer items-center space-x-1 md:space-x-2', {
```

File: src/components/Actions/index.tsx [hunk 2]
```diff
@@ -28,48 +30,60 @@ export const PostActions = memo<PostActionsProps>(function PostActions({
     disablePadding = false,
 }) {
     const publicationViews = useImpressionsStore.use.publicationViews();
+    const isSmall = useIsSmall('max');
```
Comment (guanbinrui) [inline src/components/Actions/index.tsx:33]: Recommend use `useIsMedium`

Comment (swkatmask) [reply src/components/Actions/index.tsx:33]: Why? Not show views on mobile only.

Comment (guanbinrui) [reply src/components/Actions/index.tsx:33]: useIsSmall('max') === !useIsMedium()

Comment (guanbinrui) [reply src/components/Actions/index.tsx:33]: useIsSmall('max') only needed when a rare case.

**Pull request merged**